definitions/scenarios/upgrade_to_capsule_6_16.rb

```diff
@@ -94,6 +94,7 @@ class PostUpgradeChecks < Abstract
     def compose
       add_steps(find_checks(:default))
       add_steps(find_checks(:post_upgrade))
+      add_step(Procedures::Pulpcore::ContainerHandleImageMetadata)
```

Comment:
For Satellite and Capsule you opted to put this before check for reboot and in the foreman upgrade it's after that check. I think we should be consistent.